src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -1,15 +1,24 @@
 /* eslint-disable react/prop-types */
 import React, { useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
-import { connectStateResults } from 'react-instantsearch-dom';
+import { connectStateResults, connectPagination } from 'react-instantsearch-dom';
 import { DataTable, Toast } from '@edx/paragon';
 
 import BulkEnrollmentModal from '../../containers/BulkEnrollmentModal';
+import StatusAlert from '../StatusAlert';
+import LoadingMessage from '../LoadingMessage';
 
 const emptyCourseResults = () => <div>No Courses found for this Enterprise</div>;
+const ERROR_MESSAGE = 'An error occured while retrieving data.';
+export const NO_DATA_MESSAGE = 'There are no results.';
```

Comment:
DOUBLE CHECK: I am not 100% if we consider this short microcopy, shall we drop the period? https://openedx.atlassian.net/wiki/spaces/BPL/pages/1966375321/Microcopy+writing+guidelines

Reply:
Oh, yeah, it kind of is. Will do.

Reply:
Yeah, I'd say drop the period here :)